lib/pluck_map/attribute.rb

```diff
@@ -1,15 +1,20 @@
 module PluckMap
   class Attribute
-    attr_reader :id, :model, :selects, :name, :value, :block
+    attr_reader :id, :model, :selects, :name, :alias, :value, :block
     attr_accessor :indexes
 
     def initialize(id, model, options={})
       @id = id
       @model = model
       @selects = Array(options.fetch(:select, id))
       @name = options.fetch(:as, id)
+      @alias = name.to_s.tr("_", " ")
```

Comment:
There's no need for `.tr("_", " ")` (I think this might've been a vestigial convenience for Members' version of the `to_csv` presenter, but it's inconsequential because we're not using alias in PluckMap's `to_csv.rb` and `as:` provides a way for clients to specify arbitrary human-readable column names); so when you move this to `Presenter#selects`, we can just use `Attribute#name` and don't have to reintroduce `#alias`.